.github/workflows/java-maven-openjdk11-dependency-submission.yml

```diff
@@ -8,31 +8,19 @@ on:
           The type of machine to run the job on. Must be provided as a stringified list (e.g. `runs-on: '["ubuntu-latest","self-hosted"]'`)
         required: true
         type: string
+        deprecationMessage: This workflow is deprecated in favor of java-maven-openjdk-dependency-submission
       directory:
         description: 'The directory that contains the pom.xml that will be used to generate the dependency graph from'
         default: '.'
         required: false
         type: string
+        deprecationMessage: This workflow is deprecated in favor of java-maven-openjdk-dependency-submission
```

Comment:
I don't think the `deprecationMessage` is working: 
![image](https://github.com/coveo/public-actions/assets/54333869/a56eee7b-4f33-4cf2-9fa6-549b9397e664)

Reply:
https://github.com/coveo/public-actions/pull/17